platform/golden/debian-11.7.pkr.hcl

```diff
@@ -0,0 +1,55 @@
+# This file was autogenerated by the 'packer hcl2_upgrade' command. We
```

Comment:
can you delet elines 1-17? Is this the desired hcl file?

Reply:
I'm going to remove that file all together. bullseye.pkr.hcl is the "right debian 11.7 packer for VirtualBox"